doc/source/serve/quickstart.rst

```diff
@@ -0,0 +1,45 @@
+Ray Serve: Scalable and Programmable Serving
+============================================
+
+.. image:: logo.svg
+    :align: center
+
+Serve is a serving library built on top of ray. It is easy-to-use and flexible.
+
+- Serve allows you to write *stateless* logic in serve functions and *stateful* logic in serve actors.
```

Comment:
try not to forward reference (don't mention concepts that I don't know about) - for example, what is a serve function?